app/controllers/api/v1/accounts/user_notes_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::UserNotesController < Api::BaseController
+  include Authorization
+
+  before_action -> { doorkeeper_authorize! :write, :'write:accounts' }
+  before_action :require_user!
+  before_action :set_account
+
+  def create
+    if params[:comment].blank?
+      UserNote.where(account: current_account, target_account: @account).destroy_all
```

Comment:
I don't know, it would only be used here. The reason I did this was basically to not have to discriminate between the case there exist one such record and the case where it doesn't exist, but that can be written differently with `find_by` and the safe navigation operator.